candig/server/static/src/custom_visualization.js

```diff
@@ -127,24 +128,26 @@ $("#table2").off("change").change(function() {
 
 $("#adv1_confirm").off('click').click(function() {
     document.getElementById("adv1").innerHTML = '<div class="loader_bar"></div>';
-    countRequest($("#table1").val(), [$("#key1").val()], datasetId).then(function(response) {
+    countRequest($("#table1").val(), [$("#key1").val()], datasetId).then(function(data) {
+        let response = JSON.parse(data)["results"][$("#table1").val()][0];
         if (response[$("#key1").val()] == undefined) {
             document.getElementById("adv1").innerHTML = "<p class='noPermission'>No data available</p>";
         } else {
             var selectedKey = $("#key1").val();
-            singleLayerDrawer("adv1", $("#type1").val(), "Distribution of " + splitString(selectedKey), currentDatasetName + " " + splitString($("#table1").val()), response[$("#key1").val()])
+            singleLayerDrawer_cv("adv1", $("#type1").val(), "Distribution of " + splitString(selectedKey), currentDatasetName + " " + splitString($("#table1").val()), response[$("#key1").val()])
         }
     })
 });
 
 $("#adv2_confirm").off('click').click(function() {
     document.getElementById("adv2").innerHTML = '<div class="loader_bar"></div>';
-    countRequest($("#table2").val(), [$("#key2").val()], datasetId).then(function(response) {
+    countRequest($("#table2").val(), [$("#key2").val()], datasetId).then(function(data) {
+        let response = JSON.parse(data)["results"][$("#table2").val()][0];
```

Comment:
This is changed because the `countRequest` returns full response.. so an additional step is needed here to parse out the results..